app/src/main/java/org/schabi/newpipe/settings/ContentSettingsFragment.java

```diff
@@ -207,6 +220,29 @@ private void exportDatabase(String path) {
         }
     }
 
+    private void saveSharedPreferencesToFile(File dst) {
```

Comment:
Why can't we save `/shared_prefs/org.schabi.newpipe.debug_preferences.xml` directly? Is there any advantage generating by generically?